setup/tue-functions.bash

```diff
@@ -333,20 +316,15 @@ function tue-revert
 
 function tue-revert-undo
 {
-    fs=`ls $_TUE_CATKIN_SYSTEM_DIR/src`
-    for pkg in $fs
+    for pkg_dir in $_TUE_CATKIN_SYSTEM_DIR/src/*/
     do
-        pkg_dir=$_TUE_CATKIN_SYSTEM_DIR/src/$pkg
+        pkg=$(basename $pkg_dir)
 
-        if [ -d $pkg_dir ]
+        if [ -f .do_not_commit_this ]
```

Comment:
Again, file could be anywhere. So this function would break.

Reply:
Well spotted!